fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -91,23 +216,56 @@ private fun MultiSelectBanner() {
 @LightDarkPreview
 @Composable
 private fun TabsTrayBannerPreview() {
+    TabsTrayBannerPreviewRoot(
+        selectedPage = Page.PrivateTabs,
+        normalTabCount = 5,
+    )
+}
+
+@LightDarkPreview
+@Composable
+private fun TabsTrayBannerInfinityPreview() {
+    TabsTrayBannerPreviewRoot(
+        normalTabCount = TabCounter.MAX_VISIBLE_TABS + 1,
+    )
+}
+
+@LightDarkPreview
+@Composable
+private fun TabsTrayBannerMultiselectPreview() {
+    TabsTrayBannerPreviewRoot(
+        isInMultiSelectMode = true,
+    )
+}
+
+@Composable
+private fun TabsTrayBannerPreviewRoot(
+    isInMultiSelectMode: Boolean = false,
+    selectedPage: Page = Page.NormalTabs,
+    normalTabCount: Int = 10,
+) {
+    var selectedPageState by remember { mutableStateOf(selectedPage) }
+
     FirefoxTheme {
         Box(modifier = Modifier.background(color = FirefoxTheme.colors.layer1)) {
             TabsTrayBanner(
-                isInMultiSelectMode = false,
-                onTabPageIndicatorClicked = {},
+                isInMultiSelectMode = isInMultiSelectMode,
+                selectedPage = selectedPageState,
+                normalTabCount = normalTabCount,
+                onTabPageIndicatorClicked = { page ->
+                    selectedPageState = page
+                },
             )
         }
     }
 }
 
-@LightDarkPreview
-@Composable
-private fun TabsTrayBannerMultiselectPreview() {
-    FirefoxTheme {
-        TabsTrayBanner(
-            isInMultiSelectMode = true,
-            onTabPageIndicatorClicked = {},
-        )
-    }
+private const val DRAG_INDICATOR_WIDTH_PERCENT = 0.1f
+
+private object DisabledRippleTheme : RippleTheme {
+    @Composable
+    override fun defaultColor() = Color.Unspecified
+
+    @Composable
+    override fun rippleAlpha(): RippleAlpha = RippleAlpha(0.0f, 0.0f, 0.0f, 0.0f)
 }
```

Comment:
this was the only way to disable the ripple from `TabRow` as it is baked into the component and unable to be separately colored from the selection color.

Reply:
Weird, I definitely still see a ripple. I also see it in the XML version though so maybe it's some device specific issue, since I see your screen recording above does not have it.


https://user-images.githubusercontent.com/10427470/229877952-7527448d-8cb7-4727-ad66-125d1b236e98.mp4

Reply:
👀 👀 👀 

Weird. Yeah, it might be OS/device specific then. It doesn't show up on my Pixel 2 with Android 9 nor on the Compose preview.

Reply:
Pixel 4a on Android 13 for posterity. Recently factory reset. Not immediately seeing any system settings to disable though

Reply:
gotcha gotcha. I've created a section to the mega meta for specific areas for UX to review when that team eventually does a pass on this rewrite and added the click ripples there.